script/check-code-style.js

```diff
@@ -179,32 +209,32 @@ components().forEach(component => {
   });
 });
 
-let foundYear = false;
-const currentYear = new Date().getFullYear().toString();
-const existingYears = ['2019'];
-
-for (const pastYear of existingYears) {
-  if (pastYear === currentYear) {
-    foundYear = true;
-    break;
-  }
-}
-
-if (!foundYear) {
-  warned(
-    `Please update the copyright to contain the current year ${new Date().getFullYear().toString()}`
-  );
-}
-
-const copyright = `// Copyright ${existingYears.join(
-  ', '
-)} Amazon.com, Inc. or its affiliates. All Rights Reserved.`;
 const spdx = '// SPDX-License-Identifier: Apache-2.0';
 
+let allYears = [];
+
 allFiles().forEach(file => {
   if (file.endsWith('.d.ts') || file.includes('.DS_Store')) {
     return;
   }
+  let yearsForFileInGitHistory = [];
+  const stdout = execute(`git log --pretty=format:'%ad' --date=short '${file}'`);
```

Comment:
Using git will fail if a builder downloads the repo from GitHub.

Reply:
This script is now only run for `build:release`:
https://github.com/aws/amazon-chime-sdk-js/blob/master/package.json#L18

The intention of `build:release` is that you are preparing a commit to be checked in, in which case there is an assumption that you have a Git repo.

If you are only running demos, you will only need `npm run build` which does not call this script.